examples/project_configuration/model_chat_evaluation_project.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<p>Suggestions:</p><ul><li>highlighting that <code>dataset_d</code> is optional</li><li>removing <code>description</code> </li></ul><p> </p><p>Executing this will show the following:</p>

<pre class="ql-syntax" spellcheck="false">/usr/local/lib/python3.10/dist-packages/labelbox/schema/project.py:791: UserWarning: 
            Skipping editor setup for a chat evaluation project.
            Editor was setup automatically.
            
  warnings.warn("""
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1645/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I check the code but I think that warning is triggered no matter what it is something with the actual code